admin/language/en-gb/extension/payment/razorpay.php

```diff
@@ -112,6 +112,12 @@
 $_['text_invoice']                                      = 'Invoice Details';
 $_['text_not_select_sub_ID']                             = 'Please Select Subscription ';
 $_['text_subscription_status']                          = 'Enable Subscription in Settings.';
+$_['text_webhook_cron_header']                          = 'Set the cron job in your WHMCS site server to call the Cron URL in every 5 mins frequncy.';
+$_['text_webhook_cron_point_1']                          = 'In CLI run: <strong>crontab -e</strong>';
```

Comment:
@abdulwahidsharief , why we can't use single variable and pass the HMTL within that?

Reply:
@ramth05 we can pass. but there will be many html tags inside a single string(<ol>,<li><label>) and cron url creating in controller we need to check how we can bring it in language file

Reply:
merged all points as one and using sprintf for cron url